### PR TITLE
Scrubby scrub

### PR DESCRIPTION
**Pull request by jnschaeffer**
This PR does a few things:
- Removes `glide.yaml` and `glide.lock` and replaces them with `go.mod` and `go.sum`
- Removes kubedb tests - kubedb tests were failing to build for a number of reasons, including suspicious state from the Go checksum DB, renamed import paths, and issues with nested Azure modules
- Fixes WordPress and Helm tests to be more robust and properly leverage kubeconfig files

### CODE REVIEW

**Comment by jnschaeffer** [comment]
A future PR will also remove the `vendor` directory, but that involves so many files that I want to save that for another time.

File: scripts/helm-init.sh [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 set -exou pipefail
 
-CLUSTER=$1
-WD=$(pwd)
-kubectl create serviceaccount --namespace kube-system tiller --kubeconfig=${CLUSTER}".conf" || true
-kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller --kubeconfig=${CLUSTER}".conf" || true
+CLUSTERCONF=$1
+kubectl create serviceaccount --namespace kube-system tiller --kubeconfig=${CLUSTERCONF} || true
```
Comment (adammohammed) [inline scripts/helm-init.sh]: What's the purpose of `|| true` in these lines? If this command fails for whatever reason, the subsequent lines would also fail wouldn't they?

File: rand/random.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+func WithRandomSuffix(prefix string) (string, error) {
+	suffixBytes := make([]byte, 4)
+
+	if _, errRead := rand.Read(suffixBytes); errRead != nil {
+		return "", errRead
+	}
+
+	suffix := hex.EncodeToString(suffixBytes)
+
+	bucketName := fmt.Sprintf("%s%s", prefix, suffix)
```
Comment (adammohammed) [inline rand/random.go]: This variable name doesn't make sense.

Comment (jnschaeffer) [reply rand/random.go]: That's because I stole it from something else. Whoops!

File: scripts/helm-init.sh (hunk 1 above)
Comment (jnschaeffer) [reply scripts/helm-init.sh]: There's no real purpose, other than them already being there. Want me to remove them?

Comment (adammohammed) [reply scripts/helm-init.sh]: I think as is this kind of defeats the purpose of `set -e`

**Review by adammohammed — APPROVED** [review]
Tested, does the thing, I like it.

**Review by asauber — APPROVED** [review]
LGTM

**Pull request merged**